src/Joonasw.AspNetCore.SecurityHeaders/FeaturePolicy/Builder/IFeaturePolicyBuilder.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Joonasw.AspNetCore.SecurityHeaders.FeaturePolicy.Builder
+{
+    internal interface IFeaturePolicyBuilder<out T>
+    {
+        /// <summary>
+        /// Block all
+        /// </summary>
+        void FromNowhere();
+        /// <summary>
+        /// Allowed from current domain.
+        /// </summary>
+        /// <returns></returns>
+        T FromSelf();
+        /// <summary>
+        /// Allowed from anywhere, except
+        /// data:, blob:, and filesystem: schemes.
```

Comment:
Mention about the other schemes is probably not necessary in the case of feature policy. "Allowed from any domain" would be good.